tests/completion/lib.exp0

```diff
@@ -0,0 +1,69 @@
+# -*- tcl -*-
```

Comment:
this is *very* similar to `tests/main/complete/lib.exp0`; work for another day to DRY it.